build.xml

```diff
@@ -6,13 +6,17 @@
   </description>
 
   <!-- set global properties for this build -->
+  <property environment="env"/>
   <property name="src" location="lib/src"/>
   <property name="test.src" location="test/src"/>
   <property name="dist" location="dist"/>
   <property name="classes" location="classes"/>
   <property name="bin" location="bin"/>
   <property name="jar.file" value="KBaseSearchEngine.jar"/>
-  <property name="jars.dir" value="/kb/deployment/lib/jars"/>
+  <condition property="jars.dir" value="${env.JARS_DIR}" else="/kb/deployment/lib/jars">
+    <isset property="env.JARS_DIR" />
+  </condition>
+  <!-- <property name="jars.dir" value="/kb/deployment/lib/jars"/> -->
```

Comment:
Will -Djars.dir on the ant command line still override /kb/deployment etc. ?

Reply:
@MrCreosote Yes, putting it in the cmd line does seem to override:

Steves-MacBook-Pro:KBaseSearchEngine sychan$ echo $JARS_DIR
/Users/sychan/src/jars/lib/jars
Steves-MacBook-Pro:KBaseSearchEngine sychan$ ant -Djars.dir="foo" war
Buildfile: /Users/sychan/src/KBaseSearchEngine/build.xml

BUILD FAILED
/Users/sychan/src/KBaseSearchEngine/build.xml:195: /Users/sychan/src/KBaseSearchEngine/foo does not exist.

Total time: 0 seconds
Steves-MacBook-Pro:KBaseSearchEngine sychan$ ant  war
Buildfile: /Users/sychan/src/KBaseSearchEngine/build.xml

compile:
     [copy] Copying 1 file to /Users/sychan/src/KBaseSearchEngine/classes
      [jar] Building jar: /Users/sychan/src/KBaseSearchEngine/dist/KBaseSearchEngine.jar

war:
    [mkdir] Created dir: /Users/sychan/src/KBaseSearchEngine/dist/lib
     [copy] Copying 34 files to /Users/sychan/src/KBaseSearchEngine/dist/lib
      [war] Building war: /Users/sychan/src/KBaseSearchEngine/dist/KBaseSearchEngine.war
   [delete] Deleting directory /Users/sychan/src/KBaseSearchEngine/dist/lib
     [copy] Copying 1 file to /Users/sychan/src/KBaseSearchEngine/scripts/webapps

BUILD SUCCESSFUL
Total time: 2 seconds
Steves-MacBook-Pro:KBaseSearchEngine sychan$